CONTRIBUTING.md

```diff
@@ -1,27 +1,28 @@
 # Contributing to opentelemetry-cpp-contrib
 
-The OpenTelemetry C/C++ special interest group (SIG) meets regularly. See the
-OpenTelemetry [community](https://github.com/open-telemetry/community#cc-sdk)
-repo for information on this and other language SIGs.
+## Introduction
 
-See the [public meeting notes](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw/edit)
-for a summary description of past meetings. To request edit access, join the
-meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-cpp).
+Welcome to the opentelemetry-cpp-contrib repository! This project is an integral part of the broader OpenTelemetry ecosystem, providing additional instrumentation and tools to enhance C/C++ observability.
 
-## Development
+We greatly appreciate any contributions, no matter the size or scope. Please feel free to reach out to the OpenTelemetry C/C++ community on [slack](https://cloud-native.slack.com/archives/C01N3AT62SJ) with questions or for assistance.
 
-TBD
+## Prerequisites
 
-### Build and Run Code Examples
+C++17 or higher
 
-TBD
+CMake 3.18 or higher
+
+Git
 
-## Pull Requests
+## Workflow
 
-### How to Send Pull Requests
+-Use feature branches when contributing
```

Comment:
The devs are supposed to work on the fork of this repo, not the feature branch of the repo.